prime-dotnet-webapi/Controllers/AuthorizedUsersController.cs

```diff
@@ -80,7 +80,7 @@ public async Task<ActionResult> GetAuthorizedUserById(int authorizedUserId)
 
         // GET: api/parties/authorized-users/5/sites
         /// <summary>
-        /// Gets a specific AuthorizedUser's sites.
+        /// Gets a health authorized sites for an authorized user.
```

Comment:
Should read "Gets health authority sites of an authorized user."

Reply:
Updated.